### PR TITLE
Permissions models

### PR DESCRIPTION
**Pull request by Azrael-git**
Trello card: https://trello.com/c/9XZQjViZ/57-permissions-model
1. Added custom permission model
2. Crontab for mailing

### CODE REVIEW

**Comment by Azrael-git** [comment]
Fixed Leonid's very bad boy прокол'с

File: api/authentication/models.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def create_user(self, email, password=None, **extra_fields):
         return self._create_user(email, password, **extra_fields)
 
     def create_superuser(self, email, password, **extra_fields):
-        extra_fields.setdefault('is_staff', True)
+        extra_fields.setdefault('is_staff', False)
```
Comment (pavlo-mk) [inline api/authentication/models.py]: `superuser` should have `is_staff True`. Its my opinion, maybe in task we have this option.

**Review by pavlo-mk — CHANGES_REQUESTED** [review]
I requested small change. We use `AUTH_USER_MODEL = 'authentication.StdUser'` and `create_superuser` should have `is_staff` because you cant create super user via manage.py command with access to django admin. All super users will not have access to django admin, or you really need this specific user type?

**Comment by pavlo-mk** [comment]
and please add **Description** for this PR ^^

File: api/authentication/models.py (hunk 1 above)
Comment (Azrael-git) [reply api/authentication/models.py]: I've set to True, we will create another field for moderator and function create_moderator()

**Pull request merged**